docs/en/about-appium/intro.md

```diff
@@ -2,15 +2,15 @@
 
 Appium is an open-source tool for automating native,
 mobile web, and hybrid applications on iOS and Android platforms.
-**Native apps** are those written using the iOS or Android SDKs. **Mobile web apps** are web apps accessed using a mobile browser (Appium supports
+**Native apps** are those written using the iOS, Windows or Android SDKs. **Mobile web apps** are web apps accessed using a mobile browser (Appium supports
```

Comment:
I think we ought to be consistent in the ordering of these references. `iOS, Android, and Windows` makes the most sense for me.